wallet/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "soroban-wallet-contract"
+version = "0.0.0"
+authors = ["Stellar Development Foundation <info@stellar.org>"]
+license = "Apache-2.0"
+edition = "2021"
+publish = false
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+doctest = false
+
+[features]
+testutils = ["soroban-sdk/testutils", "soroban-token-contract/testutils"]
+
+[dependencies]
+soroban-sdk = "0.0.4"
+soroban-auth = "0.0.4"
+soroban-token-contract = { git = "https://github.com/stellar/soroban-token-contract", rev = "46ca69b8", default-features = false }
```

Comment:
Before we can merge this contract we need to update how it interacts with tokens by importing the `soroban-token-spec.wasm` for the client, and creating and using the built-in token contract.

Reply:
Updated the contract to match the new SDK and token contract.